readme.md

```diff
@@ -27,4 +27,8 @@ db.write(key, val, function (err) {
 db.delete(key, function (err) {
   // deletes `key` key from data.json
 })
+
+db.update(key, val, function (err) {
+  // Updates the key with the value using `xtend` for objects
```

Comment:
Should probs document this method might have data races, given we read async and then write async